packages/vue-instantsearch/src/components/Autocomplete.vue

```diff
@@ -20,7 +20,7 @@
 </template>
 
 <script>
-import { connectAutocomplete } from 'instantsearch.js/es/connectors';
+import { connectAutocomplete } from 'instantsearch.js/es/connectors/index.umd';
```

Comment:
hmm, pretty annoying this is needed, but no way around for now as we still want regular esm users to have access to chat